### DISCUSSION TITLE
Inconsistent Selected Properties Among Site Collection Administrators

### DISCUSSION THREAD

**Discussion by Killerkadoogin** [microsoft-search/pnp-modern-search#3704] [Q&A]
Working with another site collection administrator, when we try adding Selected Properties in the PnP Search web part, we both see a different list of options to select on the same web part. My associate sees all of the Managed Properties I'd mapped to indexed columns. I don't. I've assigned the indexed column "Language" (ows_Language in Crawled Properties) to "Refinablestring14" in Managed Properties. 48 hours later, my associate sees Refinablestring14, but I don't. When my associate selects the missing properties, I can see them, but not until then.

This is what I see:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/93843972/e749b6f0-f85b-4bf2-820b-dd06db55d740)

This is what my associate sees:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/93843972/38ae6588-d75a-4860-a0b9-3f5687f4c4ff)

When I logged into SharePoint Online from his laptop, the issue persisted. So, we're relatively confident that the issue is tied to my profile. Any suggestions?

**Comment by wobba** [comment]
This is a known issue as the "API" we use is not a real API. The solution is to have a hardcoded list of properties which there is an issue for when I get around to doing it. The workaround is to manually input the property you want and disregard the list.
